packages/rc/LICENSE.md

```
@@ -57,22 +54,21 @@ Repository: git+https://github.com/visulima/visulima.git
 > OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 > SOFTWARE.
 >
-> # Licenses of bundled dependencies
 >
+>
+> # Licenses of bundled dependencies
```

markdownlint-cli2 (0.23.0)

[warning] Multiple headings with the same content

(MD024, no-duplicate-heading)